web/app/js/components/util/Utils.js

```diff
@@ -24,6 +24,25 @@ export const formatLatency = m => {
   }
 };
 
+const niceLatency = l => latencyFormatter(Math.round(l));
+
+export const formatTapLatencySec = tapLatency => {
```

Comment:
This function seems generally useful, so I recommend changing the naming so that it doesn't apply strictly to tap. In fact, I'd be in favor of using this function everywhere we format latency. Maybe something like:

```
export const formatLatencyMs = m => {
  if (_.isNil(m)) {
    return "---";
  } else {
    return formatLatencySec(m / 1000.0);
  }
};

export const formatLatencySec = latency => {
  let s = parseFloat(latency);
  if (_.isNil(s)) {
    return "---";
  } else if (s === parseFloat(0.0)) {
    return "0 s";
  } else if (s < 0.001) {
    return `${niceLatency(s * 1000 * 1000)} µs`;
  } else if (s < 1.0) {
    return `${niceLatency(s * 1000)} ms`;
  } else {
    return `${niceLatency(s)} s`;
  }
};
```